notebooks/117-optimize-preprocessing/117-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>Can we do not use OMZ here?</p><p>Just download model from original source and convert it using mo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
yes, done